ComposeSample/app/build.gradle

```diff
@@ -26,7 +26,6 @@ android {
     buildTypes {
```

Comment:
You can get rid of the whole `buildTypes`. Since we don't plan to ship apps, the `buildTypes` is not needed.